.github/actions/create-jira-issues-for-community-activities/action.yml

```diff
@@ -75,23 +80,23 @@ runs:
         fields: ${{ steps.issue-details.outputs.fields }}
 
     - name: Add Remote Link to Jira Issue
-      if: steps.issue-related-pr.outputs.is-not-issue-pr == 'True'
-      shell: bash
+      if: steps.issue-related-pr.outputs.is-issue-pr == 'False'
+      shell: pwsh
       run: |
-        curl --request POST \
-          --url '${{ env.JIRA_BASE_URL }}/rest/api/3/issue/${{ steps.create-issue.outputs.issue }}/remotelink' \
-          --user '${{ env.JIRA_USER_EMAIL }}:${{ env.JIRA_API_TOKEN }}' \
-          --header 'Accept: application/json' \
-          --header 'Content-Type: application/json' \
+        curl --request POST `
+          --url '${{ env.JIRA_BASE_URL }}/rest/api/3/issue/${{ steps.create-issue.outputs.issue }}/remotelink' `
+          --user '${{ env.JIRA_USER_EMAIL }}:${{ env.JIRA_API_TOKEN }}' `
+          --header 'Accept: application/json' `
+          --header 'Content-Type: application/json' `
           --data '{
-            "object": {
-                "url":"${{ steps.issue-details.outputs.link }}",
-                "title":"GitHub resource"
-            }
-        }'
+                \"object\": {
```

Comment:
Why is this backslashed now?

Reply:
It's necessary in PS.

Reply:
Well, as long as it works...

Reply:
It does, and it certainly didn't work without it :D.